report_templates/ansible_inventory.erb

```diff
@@ -0,0 +1,144 @@
+<%#
+name: Ansible Inventory
+description: This template has a special meaning and should not be renamed.
+snippet: false
+template_inputs:
+- name: Organization
+  required: true
+  input_type: user
+  description: Whether should be Organization included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: Location
+  required: true
+  input_type: user
+  description: Whether should be Location included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: Host Group
+  required: true
+  input_type: user
+  description: Whether should be Host Group included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: Host Collections
+  required: true
+  input_type: user
+  description: Whether should be Host Collections included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: IPv4
+  required: true
+  input_type: user
+  description: Whether should be IPv4 included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: IPv6
+  required: true
+  input_type: user
+  description: Whether should be IPv6 included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: no
+- name: Subnet
+  required: true
+  input_type: user
+  description: Whether should be Subnet included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: Smart Proxies
+  required: true
+  input_type: user
+  description: Whether should be Smart Proxies included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: yes
+- name: Content Attributes
+  required: true
+  input_type: user
+  description: Whether should be Content Attributes included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: no
+- name: Facts
+  required: true
+  input_type: user
+  description: Whether should be Facts included.
+  advanced: false
+  value_type: plain
+  options: "yes\r\nno"
+  default: no
+model: ReportTemplate
+require:
+- plugin: foreman_ansible
+  version: '>= 3.1.0'
```

Comment:
:warning: This is wrong, it can not be seeded to the foreman. Or am I missing something?

Reply:
Fixed already

Reply:
Oh, right. This is the old PR.

This was fixed in https://github.com/theforeman/community-templates/commit/34f4357d566f313eccdf8af1b46ef5598bb7e265